contracts/contracts/Subsidy.sol

```diff
@@ -16,13 +16,16 @@ import { VkRegistry } from "./VkRegistry.sol";
 /// are correct. It is also used to update the subsidy commitment if the
 /// proof is valid.
 contract Subsidy is Ownable, CommonUtilities, Hasher, SnarkCommon {
-  uint256 public rbi; // row batch index
-  uint256 public cbi; // column batch index
+  // row batch index
+  uint256 public rbi;
+  // column batch index
+  uint256 public cbi;
+
   // The final commitment to the state and ballot roots
   uint256 public sbCommitment;
   uint256 public subsidyCommitment;
 
-  uint8 public constant treeArity = 5;
+  uint8 internal constant TREE_ARITY = 5;
```

Comment:
Thinking about that `TREE_ARITY` is called so many times in different contracts, and they should keep the same. Should we put it in somewhere and import it in these contracts?

Reply:
mm maybe we could put it inside another contract shared by all of them though I think that would bloat things up a little bit 🤔 was looking and couldn't find a contract that was already in use by all of them (MACI, Poll, MP, Tally, etc) where we could add this constant..

Reply:
Is that a good idea to have a contract holding the constants? or we just keep status quo?

Reply:
mm we have `SnarkConstants` holding some constants, though I'm not sure if it's a widely used pattern.. to avoid bloating contracts even more I would just keep this one constant inside each contract, at least for now

Reply:
Then when it comes to deployment, if a user wants to alter the TREE_ARITY, they currently need to navigate through every contract file to modify the value. What if we convert it into an environmental variable for the sake of convenience during deployment?

Reply:
Fair point, though they would also need to change the type of `AccQueue` in use, which currently is hardcoded to a quinary tree everywhere, mm 🤔

Reply:
useful discussion here, there's definitely some more simplifying we can do given we always use quinary trees

Reply:
by converting to env variable, you mean that it should be a value passed in the constructor when deploying the contracts?

Reply:
yes... but not sure if this is a good idea, just wanna reduce the risk that if some users deploy it and miss to change one of them?

Reply:
let's write this down and discuss together at the next team meeting? from next week we'll have a smart contract developer to ask all SC questions to 😈